webapp/certified/views.py

```diff
@@ -358,6 +360,12 @@ def certified_home():
         # Pagination
         total_results = models_response["meta"]["total_count"]
 
+        # Add static vendor data
+        with open("webapp/certified/vendors_data.yaml") as vendors_data:
+            vendors_strip = yaml.load(
```

Comment:
Feel free to suggest a better name for this variable, I couldn't come up with anything else quickly, we are using the word "vendor" in so many variations...

Reply:
Maybe `vendors_info`? can we not use `vendors_data` as you've already used it?